peer/network.go

```diff
@@ -357,6 +428,13 @@ func (n *network) SetRequestHandler(handler message.RequestHandler) {
 	n.requestHandler = handler
 }
 
+func (n *network) SetCrossChainRequestHandler(handler message.CrossChainRequestHandler) {
+	n.lock.Lock()
+	defer n.lock.Unlock()
+
+	n.crossChainRequestHandler = handler
```

Comment:
could we either:
a) remove this in this PR since the VM does not handle incoming messages, or
b) complete the feature of the VM receiving messages

I would prefer (b) for a more feature complete PR.

Reply:
I held off on (b) because I  am unsure what message type we are going to accept for subnet-evm and how we are going to handle the message. I will remove and go with option (a)